sqlite/src/func.c

```diff
@@ -2254,7 +2254,21 @@ static void groupConcatStep(
   if( pAccum ){
     sqlite3 *db = sqlite3_context_db_handle(context);
     int firstTerm = pAccum->mxAlloc==0;
+#if defined(SQLITE_BUILDING_FOR_COMDB2)
+    struct sql_thread *thd = pthread_getspecific(query_info_key);
+    struct sqlclntstate *clnt = thd!=NULL ? thd->clnt : NULL;
+
```

Comment:
FYI, for these two lines, which are a commonly used pattern, I created a function get_sql_clnt().

Reply:
Fixed, thank you.